src/uiTest/resources/UIScenarioSuite.conf

```diff
@@ -0,0 +1,16 @@
+probe {
+  driver {
```

Comment:
One last solution could be using extracting common.conf and using `include "common"` from both `UI*Test.conf`... if not then leave as is

Reply:
it doesn't work like that sadly